backend/server.js

```diff
@@ -322,6 +392,8 @@ sbp('sbp/selectors/register', {
         }))
       }
       await sbp('chelonia.db/delete', kvIndexKey)
+      await sbp('backend/server/removeIndirectResourcesIndex', cid)
+      await sbp('chelonia.db/delete', `_private_indirectResources_${cid}`)
```

Comment:
I find it odd that a selector called `backend/server/removeIndirectResourcesIndex` wouldn't remove `_private_indirectResources_${cid}`...

Could you either rename the selector or move the call to `delete` into there?

Reply:
`backend/server/removeFromIndirectResourcesIndex`?